### PR TITLE
7 fortran test syntax

### PR DESCRIPTION
**Pull request by connoraird**

### CODE REVIEW

File: episodes/3-fortran-unit-test-syntax.md [hunk 1]
```diff
@@ -0,0 +1,310 @@
...
+
+## Let's dive into the syntax
+
+We will use the game of life example from challenge 1 of the last episode to highlight the difference in
+syntax between the three frameworks.
+
+
+### Define types to act as test parameters (and test case for pfunit)
+
+This step is similar for all three frameworks and uses standard Fortran syntax to define a [derived type](https://fortran-lang.org/learn/quickstart/derived_types). The key differences are...
```
Comment (dpshelio) [inline episodes/3-fortran-unit-test-syntax.md:64]:
```suggestion
This step is similar for all three frameworks and uses standard Fortran syntax to define a [derived type](https://fortran-lang.org/learn/quickstart/derived_types). The key differences are:
```

File: episodes/3-fortran-unit-test-syntax.md [hunk 2]
```diff
@@ -0,0 +1,310 @@
...
+type, extends(ParameterizedTestCase) :: my_test_case
+    type(my_test_params) :: params
+end type my_test_case
+```
+
+### Define a test suite (collection of tests) to be returned from a procedure
+
+In this section we define our suite of tests to test the unit in question. This can return a single test but
+it's likely that there are multiple scenarios and edge cases we would like to test. Therefore, we return an
+array of tests rather than a single test.
```
[the remark this answers is not in the document]
Comment (connoraird) [reply episodes/3-fortran-unit-test-syntax.md:119]: Yes it is
